source/features/remove-projects-tab.tsx

```diff
@@ -31,40 +29,22 @@ const addNewProjectLink = onetime(() => {
 			New project
 		</a>
 	);
-});
-
-async function init(): Promise<false | void> {
-	await elementReady(`
-		.orghead + *,
-		.repohead + *,
-		.user-profile-nav + *
-	`); // Wait for the tab bar to be loaded
+}
 
-	const projectsTab = select([
-		'[data-hotkey="g b"]', // In organizations and repos
-		'.user-profile-nav [href$="?tab=projects"]' // In user profiles
-	]);
+async function removeProjectsTab(): Promise<void | false> {
+	const projectsTab = await getProjectsTab();
 
 	if (!projectsTab) {
 		// Projects aren't enabled here
-		return;
+		return false;
 	}
 
-	addNewProjectLink();
-
-	// If there's a settings tab, the current user can disable the projects,
-	// so the tab should not be hidden
-	if (select.exists([
-		'.js-repo-nav [data-selected-links^="repo_settings"]', // In repos
-		'.pagehead-tabs-item[href$="/settings/profile"]' // In organizations
-	])) {
-		return;
+	// Dont run if the counter is 0. (When it does not exists it will converted to a 0)
+	if (Number(select('.Counter', projectsTab)?.textContent) !== 0) {
```

Comment:
Actually, wrong comment and wrong code. "Nothing is converted to zero" and if there's no counter, it will `return`